### ISSUE TITLE
Screen Summary Updates

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#113]
Can we update the entire screen summaries to be like this:

**Explore Screen**
**P1: The interactive Explore Screen changes as you play with it. It has a Play Area and a Control Area.**

P2: NO CHANGES

**P3: Move hands individually or at the same time to explore the challenge.**

P4: STATIC KB shortcuts hint (no change).

**Create Screen**
**P1: The interactive Create Screen changes as you play with it. It has a Play Area and a Control Area.**

**P2: On this screen you can set your own challenge. The Play Area is an open space with two movable hands. Hands can be moved individually or simultaneously or with ratio locked to explore a challenge. There are options to adjust the look and feel of the open space that may help you master the challenges that you set. The Control Area has a button to reset the sim.**

**P3: Explore current challenge or set your own.**

P4: STATIC KB shortcuts hint (no change).

**Comment by terracoda** [comment]
Bolded text is the new text. Please don't copy in pseudo code :-)

**Comment by zepumph** [comment]
P1 changes came from common code last night. See https://github.com/phetsims/joist/issues/642

**Comment by zepumph** [comment]
Implemented above, @terracoda please review.

**Comment by Matthew-Moore240** [comment]
Couple of things @zepumph 
Explore screen doesn't seem to have P3 implemented which the help text for the scene summary

Create screen doesn't have P2 or P3 implemented

**Comment by zepumph** [comment]
>Explore screen doesn't seem to have P3 implemented which the help text for the scene summary
Create screen doesn't have P2 or P3 implemented


hmmm. This is what I see in https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-dev.46/phet/ratio-and-proportion_a11y_view.html:

Explore with p3 highlighted:
![image](https://user-images.githubusercontent.com/6856943/88982632-4c388300-d275-11ea-96d5-30454969ce14.png)

Create with p3 highlighted:
![image](https://user-images.githubusercontent.com/6856943/88982665-61adad00-d275-11ea-818f-cd5c6f430167.png)



Maybe something was cached?

**Comment by Matthew-Moore240** [comment]
I was on the wrong dev version totally my bad. These all look great! Closing
